health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/MyCampaignNew.js

```
@@ -0,0 +1,60 @@
+
+    window.addEventListener("HCM_TIMELINE_POPUP_CHANGE", handleStorageChange);
+
+    return () => {
+      window.removeEventListener("HCM_TIMELINE_POPUP_CHANGE", handleStorageChange);
+    };
+  }, [Digit.SessionStorage.get("HCM_TIMELINE_POPUP")]);
+
+  return (
+    <React.Fragment>
+      <div className="digit-inbox-search-wrapper">
+        <InboxSearchComposer configs={config} showTab={true} tabData={tabData} onTabChange={onTabChange}></InboxSearchComposer>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)